osu.Game.Tests/Visual/Gameplay/TestSceneStoryboardSamplePlayback.cs

```diff
@@ -31,10 +41,13 @@ private void load(OsuConfigManager config)
             backgroundLayer.Add(new StoryboardSampleInfo("Intro/welcome.mp3", time: 0, volume: 20));
         }
 
+        [SetUp]
+        public void SetUp() => Schedule(() => storyboardMods = Array.Empty<Mod>());
```

Comment:
Why is this scheduled?

Reply:
Was under the habit of scheduling all `SetUp` calls, will remove.